components/datadog/fakeintake/connectionExporter.go

```diff
@@ -9,37 +9,37 @@ import (
 )
 
 const (
-	stackKey = "fakeintake-url"
+	stackKey = "fakeintake-ip-address"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should this be `fakeintake-host` ? If I remember well, the exported value is the ip address plus the port

Reply:
Definitively !
In fact, we currently put an IP address: https://github.com/DataDog/test-infra-definitions/blob/ee56648ad2565da9a046af34a93de24d4cd9a09b/scenarios/aws/ecs/fakeintake.go#L17

That’s why I initially replaced `url` by `ip address`.
But Vincent made me notice that, in practice, it could also be a DNS host name: https://github.com/DataDog/test-infra-definitions/pull/133#discussion_r1221696948.
Hence the renaming to `host`: https://github.com/DataDog/test-infra-definitions/pull/133/commits/30b5719537cfb2b960ccdc81bdd2b7f0ce971ff8.
But I forgot this one.

Thanks for having caught it !!!

I made the change: https://github.com/DataDog/test-infra-definitions/pull/133/commits/ee56648ad2565da9a046af34a93de24d4cd9a09b#diff-aa24a3296acc5428619072ea3e9efb11e4424e206c7d0803f9a610461f67c51bR12